deploy/helm/Chart.yaml

```diff
@@ -6,12 +6,12 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
```

Comment:
no need to update chart and app version.
its going to be updated before a new release of starboard is published

Reply:
@chen-keinan Do you want me to do anything here? Or is that just for my info will be overwritten anyway while publishing a new release/tag?

Reply:
No need for you to do anything.